### DISCUSSION TITLE
Order segments in order of start time?

### DISCUSSION THREAD

**Discussion by LeoArtaza** [mifi/lossless-cut#1698] [General]
Edit: Found the answer right away, sorry. There's an option to sort segment by start time in Edit -> Segments. I looked for it in Tools, or in export options but not there.
Idea 1: this makes me think that it would be cool to have a search bar for features or settings at the top.
Idea 2: it would be nice to not need to merge the videos first, but rather read N amount of files at once as chapters, and then start segmenting right away.

Original:
Hi! First of all for this great piece of software, I use it on the daily to clip my gaming videos.

I was wondering if there's an option to automatically sort the segments by start time? What happened to me in a recent project was that I first merged like 10 videos, then I started creating segments in the important parts, but the new segments I created at the beginning were placed at the end in order. Now I'd have to manually sort them out.

**Comment by mifi** [comment]
Hi.

> Idea 1: this makes me think that it would be cool to have a search bar for features or settings at the top.

I agree that would be nice, adding here #189 - note that you can already search for actions in the "keyboard shortcuts" panel

> Idea 2: it would be nice to not need to merge the videos first, but rather read N amount of files at once as chapters, and then start segmenting right away.

now sure what you mean by this exactly.

**Reply by LeoArtaza** [reply]
Hi, thanks for considering the first idea!

What I mean by the 2nd one is the following (I will use my own use case as example): suppose I have 10 gaming clips, each lasts 2 minutes but I only need approx. 10 seconds of each which I will the merge into a compilation. What I do right now is either
1. Clip them separately, then merge them.
2. Merge the 2 minute clips together into a 20 minute clip, open it, and clip+merge it.

I really prefer the workflow of the 2nd one, but it's a bit cumbersome having to merge them into a big video file first, reopen, clip, then delete the unclipped merged file manually. What would be cool is to drag multiple video files, and visualize them like we would see a merged file with chapters (like this)

![image](https://github.com/mifi/lossless-cut/assets/57342159/7d88cce4-64ea-408c-8b62-065dc5b455c2)

without having to actually merge the video files first which takes a bit of time, instead just drag and drop multiple video files instantly into this view.

**Reply by mifi** [reply]
Ah, I believe what you're asking for is #976
